install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1688,10 +1693,187 @@ dashboards:
   labelValue: "1"
   annotations: {}
 
-# -- Configure Istio proxy options.
+# -- Configure Cilium Proxy options.
 proxy:
+  # Enable Cilium Proxy in standalone DaemonSet.
+  enabled: false
+
+  # Location of unix sockets for communication between Agent & Proxy.
+  socketDir:
+    # Socket directory within the proxy & agent containers
+    container: "/var/run/proxy"
+    # Socket directory within the host
+    host: "/var/run/cilium-proxy"
+
+  # Time in seconds after which a TCP connection attempt times out
+  connectTimeout: 2
```

Comment:
as the attribute name didn't contain unit (e.g. connectTimeoutSeconds), we normally have value as 2s instead. This will help to remove `s` append in bootstrap json file.

Reply:
these config properties already exist in the cilium agent. i just added them to the helm chart to be able to pass the same values to the bootstrapconfig and the agent configmap.

```
  proxy-connect-timeout: {{ .Values.proxy.connectTimeout | quote }}
  proxy-max-requests-per-connection: {{ .Values.proxy.maxRequestsPerConnection | quote }}
  proxy-max-connection-duration-seconds: {{ .Values.proxy.maxConnectionDuration | quote }}
```

and `proxy-connect-timeout` already doesn't contain the "in seconds" in its name. therefore if i change something, i tend to fix the name in the helm chart and add `Seconds` at the end. would this be ok for you? this way we still have to add the `s` as suffix in the bootstrap - but we have to add or remove it somewhere (if we don't want to rename the existing cilium config property or change its type)

Reply:
@sayboras added suffix `Seconds` to `connectTimeout` & `maxConnectionDuration`